src/App.js

```diff
@@ -1,9 +1,17 @@
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useEffect } from 'react';
 import Categories from './components/Categories';
 import Navbar from './components/Navbar';
 import Home from './components/Home';
+import { fetchBooks } from './redux/books/booksSlice';
 
 function App() {
+// const { books, isLoading, error } = useSelector((state) => state.book);
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
thanks for reminder, I am working on resolving it